src/winston.utilities.ts

```diff
@@ -1,14 +1,26 @@
-import { green, yellow } from 'colors/safe';
 import { Format } from 'logform';
+import bare from 'cli-color/bare';
+import clc from 'cli-color';
 import { format } from 'winston';
 import safeStringify from 'fast-safe-stringify';
 
+const nestLikeColorScheme: Record<string, bare.Format> = {
+  info: clc.greenBright,
+  error: clc.red,
+  warn: clc.yellow,
+  debug: clc.magentaBright,
+  verbose: clc.cyanBright,
+};
+
 const nestLikeConsoleFormat = (appName = 'NestWinston'): Format => format.printf(({ context, level, timestamp, message, ...meta }) => {
-  return `${green(`[${appName}]`)} ` +
-         `${yellow(level.charAt(0).toUpperCase() + level.slice(1))}\t` +
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Without this, eslint warns `Unnecessary conditional, value is always truthy.` Is it acceptable?

Reply:
Yes, very good solution!